R/step_climate.R

```diff
@@ -121,6 +132,11 @@ step_climate <-
     n_outcomes <- sum(recipe$var_info$role == "outcome")
     time_type <- rlang::arg_match(time_type)
     edf_time_type <- attr(recipe$template, "metadata")$time_type
+    if (time_type == "detect") {
+      time_type <- edf_time_type
+    } else {
+      edf_time_type <- attr(recipe$template, "metadata")$time_type
+    }
```

Comment:
Do we need the "else" branch? It doesn't seem necessary given line 134.
```suggestion
    if (time_type == "detect") time_type <- edf_time_type
```